index.js

```diff
@@ -36,6 +36,15 @@ function hasPragma(text) {
   return pragmas.indexOf("prettier") !== -1 || pragmas.indexOf("format") !== -1;
 }
 
+// Stolen from jest-docblock
+// source: https://github.com/facebook/jest/blob/80bd4cd4b6100d493290293645095506c968f6cf/packages/jest-docblock/src/index.js#L15
+const docblockRe = /^\s*(\/\*\*?(.|\r?\n)*?\*\/)/;
+function stripDocblock(text) {
```

Comment:
This seems like it could be part of `jest-docblock`. I'll open a PR there and see what they think

Reply:
PR Opened! https://github.com/facebook/jest/pull/4571

Reply:
and merged 🎉 ! Once its published I'll continue here again